account_payment_sale/models/sale_order.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2014-2020 Akretion France (https://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    payment_method_line_id = fields.Many2one(
```

Comment:
The change of model would need a computation script

Reply:
Of course !